Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+log = "0.4.22"
```

Comment:
Logger seems, unsued, please remove it.
I'm keeping the project dependency free for now.